### PR TITLE
feat: add release testing script

### LINKED ISSUE #1904: Create semi-automated release testing script
Goal is to facilitate snapshot e.g. #1903 and released package e.g. #1842  testing in as automated a fashion as possible.

### PR DESCRIPTION
**Pull request by dcroote**
Closes #1904.

### CODE REVIEW

**Review by Siegrift — APPROVED** [review]
👍 LGTM, but haven't run the script myself.

**Review by andreogle — APPROVED** [review]

File: scripts/test-release.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+  executeCommandSync('yarn deploy-requester');
+  executeCommandSync('yarn derive-and-fund-sponsor-wallet');
+  executeCommandSync('yarn sponsor-requester');
+  executeCommandSync('yarn make-request');
+  executeCommandSync('yarn make-withdrawal-request');
+
+  const httpConfig = {
+    method: 'post',
+    url: `${gatewayUrls.httpGatewayUrl}/0xfb87102cdabadf905321521ba0b3cbf74ad09c5d400ac2eccdbef8d6143e78c4`,
```
[the remark this answers is not in the document]
Comment (dcroote) [reply scripts/test-release.ts:175]: Added the origin of the endpointId, thanks

File: scripts/test-release.ts
[the remark this answers is not in the document]
Comment (dcroote) [reply scripts/test-release.ts:1]: Good idea- done!

File: scripts/test-release.ts [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  const signedConfig = {
+    method: 'post',
+    maxBodyLength: Infinity,
+    url: `${gatewayUrls.signedHttpGatewayUrl}/0xfb87102cdabadf905321521ba0b3cbf74ad09c5d400ac2eccdbef8d6143e78c4`,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    data: JSON.stringify({
+      encodedParameters:
+        '0x3173000000000000000000000000000000000000000000000000000000000000636f696e49640000000000000000000000000000000000000000000000000000626974636f696e00000000000000000000000000000000000000000000000000',
```
[the remark this answers is not in the document]
Comment (dcroote) [reply scripts/test-release.ts]: Done!

**Pull request merged**